libpod/healthcheck_windows.go

```diff
@@ -0,0 +1,19 @@
+// +build windows
```

Comment:
FYI, I believe this is not necessary when you build with a _windows.go postfix.  Similarly _linux.go and _darwain.go.